src/mono/wasm/runtime/jiterpreter-support.ts

```diff
@@ -1774,6 +1775,22 @@ export function bytesFromHex(hex: string): Uint8Array {
     return bytes;
 }
 
+export function isZeroPageReserved(): boolean {
+    // FIXME: This check will always return true on worker threads.
```

Comment:
@kg could you please add `mono_assert(!MonoWasmThreads, "FIXME: This check will always return true on worker threads.");` on some future PR. Thanks.

Reply:
[!] (plugin typescript) RollupError: @rollup/plugin-typescript TS2304: Cannot find name 'MonoWasmThreads'.
  jiterpreter-support.ts (1786:9)
  
  1786     if (MonoWasmThreads)
               ~~~~~~~~~~~~~~~

Reply:
@kg it's in a magic rollup constant module

```ts
import MonoWasmThreads from "consts:monoWasmThreads";
```